library/Icinga/Web/Cookie.php

```diff
@@ -262,4 +269,13 @@ public function setValue($value)
         $this->value = $value;
         return $this;
     }
+
+    public function forgetMe()
```

Comment:
Please add PHPDoc with the following information:

* What this method does
* How it's done
* This function alone does not clear the cookie. What else is needed?

For describing that this method returns a new instance of the class use `@return static`.